src/types/index.ts

```diff
@@ -0,0 +1,7 @@
+export * from './contacts.types';
+export * from './store.types';
+
+export type ErrorType = {message?: any};
+export type ChildrenType = {children?: any};
+
+export type UiStateType = 'INIT' | 'IN_PROGRESS' | 'FAILED' | 'SUCCESS';
```

Comment:
This won't be a proper union type. I guess we discussed this last time.

Reply:
I have some doubts on union Type. We will discuss today evening